CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Change Log
 
+## [2.0.120230525]
+- Updating to latest B2K CLI version - please refer for release notes [here](https://github.com/Azure/Bridge-To-Kubernetes/pull/275)
```

Comment:
wondering why this is a link to a pr instead of a link to the changelog in main like the other entries here. Seems better to link to the actual changelog after that pr is merged in so if you need to update the changelog in the main b2k repo you don't have to remember to update this link.

Reply:
updated. Thanks for the review.